### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KlausLoeffelmann**
Erase redundant README sections.

### CODE REVIEW

**Comment by Tanya-Solyanik** [comment]
As far as I understood Viktor's comment, 
 - [] this was a wrong readme to edit
 - [] you need to so package authoring or something
 - [x] remove versioning info

File: src/Microsoft.Windows.Compatibility/src/Microsoft.Windows.Compatibility.csproj [hunk 1]
```diff
@@ -10,8 +10,13 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageDescription>This Windows Compatibility Pack provides access to APIs that were previously available only for .NET Framework. It can be used from both .NET as well as .NET Standard.</PackageDescription>
+    <PackageReadmeFile>PACKAGE.md</PackageReadmeFile>
   </PropertyGroup>
 
+  <ItemGroup>
+      <None Include="PACKAGE.md" Pack="true" PackagePath="" />
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/Microsoft.Windows.Compatibility.csproj:17]: Indent by 2 spaces

File: src/Microsoft.Windows.Compatibility/src/PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Microsoft.Windows.Compatibility
+
+The `Microsoft.Windows.Compatibility` package provides Windows-specific APIs to help you port your .NET Framework applications to .NET Core 2.0+, .NET 5+ or .NET Standard. This package offers a smoother transition for those looking to modernize their applications without losing access to familiar Windows functionalities.
+
+## Getting Started
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/PACKAGE.md:5]: Move the link to nuget.org to this section?

File: src/Microsoft.Windows.Compatibility/src/PACKAGE.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- [Use the Windows Compatibility Pack to port code - .NET](https://learn.microsoft.com/dotnet/core/porting/windows-compat-pack)
+- [Announcing the Windows Compatibility Pack for .NET Core](https://devblogs.microsoft.com/dotnet/announcing-the-windows-compatibility-pack-for-net-core/)
+- [Installing NuGet client tools | Microsoft Learn](https://learn.microsoft.com/nuget/consume-packages/install-use-packages-nuget-cli)
+
+## Feedback
+
+We value your feedback! Here are ways to get in touch with us:
+
+- Open an issue on our [GitHub repository](https://github.com/dotnet/runtime/issues)
+- Reach out on Twitter with the hashtag #dotnet
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/PACKAGE.md:91]: Link for twitter?

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 4]
```diff
@@ -1,108 +1,56 @@
...
-- Reach out on Twitter with the hashtag #dotnet
-- Join our Discord channel: [dotnet/Discord](https://discord.com/invite/dotnet)
+# .NET Windows Desktop Runtime
+
+This repo contains the code to build the .NET Windows Desktop Runtime for all
+supported platforms.
+
+## Getting started
+
+* [.NET 8 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:8]: Remove en-us

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 5]
```diff
@@ -1,108 +1,56 @@
...
+* [Windows Forms repository](https://github.com/dotnet/winforms)
+* [WPF repository](https://github.com/dotnet/wpf)
+
+## How to Engage, Contribute and Provide Feedback
+
+Some of the best ways to contribute are to try things out, file bugs, join in
+design conversations, and fix issues.
+
+* This repo defines [contributing guidelines](CONTRIBUTING.md) and also follows
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:17]: This link looks wrong - check the whole doc in VS Code for wrong links.

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 6]
```diff
@@ -1,108 +1,56 @@
...
+Conduct](https://dotnetfoundation.org/code-of-conduct) to define expected
+conduct in our community. Instances of abusive, harassing, or otherwise
+unacceptable behavior may be reported by contacting a project maintainer at
+conduct@dotnetfoundation.org.
+
+## License
+
+.NET (including the WindowsDesktop repo) is licensed under the [MIT license](LICENSE.TXT).
+
+## Officially Released Builds
```
Comment (Tanya-Solyanik) [inline src/Microsoft.Windows.Compatibility/src/README.md:54]: Is the empty section needed?

File: src/Microsoft.Windows.Compatibility/src/PACKAGE.md (hunk 2 above)
Comment (KlausLoeffelmann) [reply src/Microsoft.Windows.Compatibility/src/PACKAGE.md:5]: At that point, aren't folks already on that page?

File: src/Microsoft.Windows.Compatibility/src/README.md [hunk 7]
```diff
@@ -1,108 +1,52 @@
-# Microsoft.Windows.Compatibility
+# .NET Windows Desktop Runtime
```
Comment (ViktorHofer) [inline src/Microsoft.Windows.Compatibility/src/README.md:1]: Revert the changes in this  file. This is the wrong content.

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
You want to backport this and the System.Drawing.Common change one into release/8.0 to make sure that it is included in the .NET 8 release.
